tests/kernel/usage/thread_runtime_stats/src/test_thread_runtime_stats.c

```diff
@@ -18,7 +18,8 @@
 #if defined(CONFIG_RISCV)
 #define IDLE_EVENT_STATS_PRECISION 7
 #else
-#define IDLE_EVENT_STATS_PRECISION 1
+/* some system may have more clock in idle*/
+#define IDLE_EVENT_STATS_PRECISION 7
```

Comment:
The changes to this file are not related to this commit; please remove them.

Reply:
it is related, I find in QEMU platfrom, the runtime variation is bigger than real board, especially when there is a idle called.

Reply:
ok